ParkingZoneApp.Tests/Controllers/Admin/ParkingSlotControllerUnitTests.cs

```diff
@@ -246,13 +255,38 @@ public void GivenEditVMAndParkingSlotId_WhenEditPostIsCalled_ThenReturnModelErro
                     .Verify(x => x.IsUniqueNumber(editVM.ParkingZoneId, editVM.Number), Times.Once);
         }
 
+        [Fact]
+        public void GivenEditVMAndParkingSlotId_WhenEditPostIsCalled_ThenReturnModelErrorIfSlotIsInUse()
```

Comment:
why return it is grammatically wrong i think lets do like this 
```suggestion
        public void GivenEditVMAndParkingSlotId_WhenEditPostIsCalled_ThenSlotIsInUseAndModelStateIsFalseAndReturnsViewResult()
```